app/services/analytics.rb

```diff
@@ -51,7 +49,7 @@ def uuid
   INVALID_AUTHENTICITY_TOKEN = 'Invalid Authenticity Token'.freeze
   INVALID_SERVICE_PROVIDER = 'Invalid Service Provider'.freeze
   OTP_DELIVERY_SELECTION = 'OTP: Delivery Selection'.freeze
-  OTP_RESULT = 'OTP: Result'.freeze
+  MFA = :mfa
```

Comment:
It seems like a lot of our other event constants are more verbose -- maybe change this to `MULTI_FACTOR_AUTH` for consistency / clarity?